staging/src/k8s.io/apimachinery/pkg/util/validation/field/errors.go

```diff
@@ -50,7 +50,10 @@ func (v *Error) ErrorBody() string {
 		s = fmt.Sprintf("%s", v.Type)
 	default:
 		value := v.BadValue
-		if reflect.TypeOf(value).Kind() == reflect.Ptr {
+		valueType := reflect.TypeOf(value)
+		if value == nil || valueType == nil {
```

Comment:
I guess it can't. I can remove the value compare if you want, it just saved me reasoning through reflection for simple cases.